src/core/power_flow_data_wrapper.jl

```diff
@@ -2,18 +2,20 @@ mutable struct PowerFlowEvaluationData{T <: PFS.PowerFlowContainer}
     power_flow_data::T
     """
     Records which PSI keys are read as input to the power flow and how the data are mapped.
-    For `PowerFlowData`, values are `Dict{String, Int64}` mapping component name to matrix
-    index of bus; for `SystemPowerFlowContainer`, values are Dict{Union{String, Int64},
+    The Symbol is a category of data: `:active_power`, `:reactive_power`, etc. The
+    `OptimizationContainerKey` is a source of that data in the `OptimizationContainer`. For
+    `PowerFlowData`, leaf values are `Dict{String, Int64}` mapping component name to matrix
+    index of bus; for `SystemPowerFlowContainer`, leaf values are Dict{Union{String, Int64},
     Union{String, Int64}} mapping component name/bus number to component name/bus number.
     """
-    input_key_map::Dict{<:OptimizationContainerKey, <:Any}
+    input_key_map::Dict{Symbol, <:Dict{<:OptimizationContainerKey, <:Any}}
```

Comment:
Why this change?

Reply:
Separate dicts for active and reactive power, basically. There were other ways to do it but I think this is the most natural.